### ISSUE TITLE
Is this ready to work?

### ISSUE THREAD

**Issue by lunny** [thehowl/giteafmt#1]
Maybe we can use this tool on drone CI to ask PR formatted before merge.

**Comment by thehowl** [comment]
Not really. I found that this removes existing comments out of imports, so not ready yet.

**Comment by andreynering** [comment]
There's https://github.com/rogpeppe/sortimports

It just would have to be forked to sort `code.gitea.io`
